_source/user-guide/kibana/field-not-indexed.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: article
+title: Field not indexed
+permalink: /user-guide/kibana/mapping/field-not-indexed/
+flags:
+  logzio-plan: community
+tags:
+  - kibana
+contributors:
+  - shalper
+---
+
+Sometimes, it will appear that a field in Kibana is not mapped. The mapping icon will show a <i class="fas fa-question"></i> question mark indicating that the field was not mapped.
```

Comment:
```suggestion
Sometimes, it will appear that a field in Kibana is not mapped. The mapping icon will show a <i class="fas fa-question"></i> question mark indicating that the field is not mapped in Kibana. 
```